config/wom.js

```diff
@@ -1,8 +1,37 @@
 const { WOMClient } = require('@wise-old-man/utils');
-
 const { Environment } = require('../services/environment');
 
 let womClient;
+let availTokens = 100;
+let lastTokenReplenish = Date.now();
+
+const replenishTokens = () => {
+    const now = Date.now();
+    const elapsedTime = now - lastTokenReplenish;
+    const tokensToAdd = Math.floor((elapsedTime / 60000) * 100);
```

Comment:
What you could do to make this use of "100" on 3 lines more explicit, is by defining a constant on the top of this file like this:
```const WOM_RATE_LIMIT = 100;```
and then use it in the code like this:
```
const tokensToAdd = Math.floor((elapsedTime / 60000) * WOM_RATE_LIMIT);
availTokens = Math.min(availTokens + tokensToAdd, WOM_RATE_LIMIT);
```

etc. etc.